services/frontend/src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import SidebarLayout from "./SidebarLayout";
+
+const Sidebar = ({sections, containerClass}) => (
```

Comment:
I had it as `sidebar` since it's not necessarily only going to be used for the profile page, we have the project's page which uses a very similar component (plus changelog).

If you would rather just create two separate component's I can just name it `ProfileSidebar`

Reply:
Hmm, right... In that case, maybe something like `NavSidebar` or `PageSidebar` would be more appropriate.

Reply:
Sure, re-named it to NavSidebar if that helps.